webview/src/experiments/components/table/body/branchDivider/BranchDivider.tsx

```diff
@@ -18,6 +19,9 @@ export const BranchDivider: React.FC<PropsWithChildren> = ({ children }) => (
           {children}
         </div>
       </th>
+      <th colSpan={9999} className={styles.branchActions}>
```

Comment:
[Q] why do we need `9999`?

Reply:
It makes the cell take all the available space (as long as there are less than 1000 cells, I guess at that point we don't care anyway). Instead of calculating the number of cells and passing that information around as props.